R/rules-line-breaks.R

```diff
@@ -128,7 +128,7 @@ set_line_break_around_comma_and_or <- function(pd, strict) {
     (pd$token %in% ops) &
       (pd$lag_newlines > 0L) &
       (pd$token_before != "COMMENT") &
-      (lag(pd$token) != "'['")
+      !(lag(pd$token) %in% c("LBB", "'['"))
```

Comment:
Let's assign `c("LBB", "'['")` to `left_token` variable and reuse that variable?

Reply:
Because we are using this vector in more than one location.

Reply:
I see, but I'd do that in another place. Will force push...

Reply:
We can re-use it in another file and we have defined groups of tokens before top-level.